DistroLauncher/DistroLauncher.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "stdafx.h"
 
+

```

Comment:
Please remember that we want our footprint the smallest possible on upstream files.

Reply:
Fixed.